nmdc_automation/re_iding/base.py

```diff
@@ -131,6 +144,7 @@ def update_omics_processing_has_output(self, db_record: Dict,
             params = copy.deepcopy(old_do_rec)
             params.pop("id", None)
             new_do_id = self.api_client.minter("nmdc:DataObject")
+            self.id_changes.append(("nmdc:DataObject", old_do_id, new_do_id))
```

Comment:
I like the variable names, `old_do_id` and `new_do_id` (I find them to be clear).